scopes/docs/docs/docs.ui.runtime.tsx

```diff
@@ -32,7 +32,7 @@ export class DocsUI {
   private _emptyState?: ComponentType;
 
   registerEmptyState(emptyState: ComponentType) {
-    return this._emptyState = emptyState;
+    return (this._emptyState = emptyState);
```

Comment:
Please remove from this pr

Reply:
I did not do such changes. What is it? Autoformat?